portal-web/docroot/html/taglib/ui/input_localized/page.jsp

```diff
@@ -134,7 +134,7 @@
 			%>
 
 			<liferay-ui:icon-menu
-				direction="left-side"
+				direction="<%= languagesDropdownDirection %>"
```

Comment:
As we commented before just make sure that this is not breaking if no `languagesDropdownDirection` is passed

Reply:
Just tested it, if we don't pass a value for `languagesDropdownDirection` it will use the default alignment, which is to the left (as it was before these changes) 💃